examples/keras_io/tensorflow/nlp/end_to_end_mlm_with_bert.py

```diff
@@ -0,0 +1,493 @@
+"""
+Title: End-to-end Masked Language Modeling with BERT
+Author: [Ankur Singh](https://twitter.com/ankur310794)
+Converted to Keras-Core: [Mrutyunjay Biswal](https://twitter.com/LearnStochastic)
+Date created: 2020/09/18
+Last modified: 2023/09/06
+Description: Implement a Masked Language Model (MLM) with BERT and fine-tune it on the IMDB Reviews dataset.
+Accelerator: GPU
+"""
+"""
+## Introduction
+
+Masked Language Modeling is a fill-in-the-blank task,
+where a model uses the context words surrounding a mask token to try to predict what the
+masked word should be.
+
+For an input that contains one or more mask tokens,
+the model will generate the most likely substitution for each.
+
+Example:
+
+- Input: "I have watched this [MASK] and it was awesome."
+- Output: "I have watched this movie and it was awesome."
+
+Masked language modeling is a great way to train a language
+model in a self-supervised setting (without human-annotated labels).
+Such a model can then be fine-tuned to accomplish various supervised
+NLP tasks.
+
+This example teaches you how to build a BERT model from scratch,
+train it with the masked language modeling task,
+and then fine-tune this model on a sentiment classification task.
+
+We will use the Keras-Core `TextVectorization` and `MultiHeadAttention` layers
+to create a BERT Transformer-Encoder network architecture.
+
+Note: This is only tensorflow backend compatible.
+"""
+
+"""
+## Setup
+"""
+
+import os
+import re
+import glob
+import numpy as np
+import pandas as pd
+from pathlib import Path
+from dataclasses import dataclass
+
+import tensorflow as tf
+import keras_core as keras
+from keras_core import layers
+from keras_core.models import Model
+from keras_core.initializers import Constant
+from keras_core.layers import TextVectorization
```

Comment:
The 3 objects above can be used from `keras` or from `layers` directly, no need to import them